ui-admin/src/study/surveys/FormOptionsModal.tsx

```diff
@@ -60,7 +74,15 @@ export const FormOptions = ({ studyEnvContext, initialWorkingForm, updateWorking
                                 updateWorkingForm: (props: SaveableFormProps) => void
                               }) => {
   const workingForm = initialWorkingForm as Survey
-
+  const { onChange, options, selectedOption, selectInputId } =
```

Comment:
despite only having one Select, this component has a ton of configuration options, so I think the more specific naming helps a bit:

```suggestion
  const { onChange: onRecurrenceChange, options: recurrenceOptions, selectedOption: selectedRecurrenceOption, selectInputId: selectRecurrenceInputId } =
```